src/components/input/input.tsx

```diff
@@ -103,26 +113,28 @@ export class Input implements LabelableComponent, FormComponent, InteractiveComp
   @Prop() intlClear?: string;
 
   /**
-   * string to override English loading text
+   * A text label that will appear while loading for screen readers.
```

Comment:
For some of the other component consistency we were using "Accessible name that will...". Since screen readers are only one type of user, we could keep it general to ensure we're including more audiences, so maybe something like **Accessible name that will appear while loading**?